CS_Assignment_2/Question_3.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CS_Assignment_2
+{
+    internal class Question_3
+    {
+        Double id;
+        string? Name;
+        string? Education;
+
+        public override string ToString()
+        {
+            return id + " " + Name + " " +
+                   Education;
+        }
+
+        public void showStudent()
+        {
+            List<Question_3> students = new List<Question_3>
+            {
+                new Question_3{id=1,Name="Abhishek",Education="CSE" },
```

Comment:
you should not create object of own class.

Reply:
updated changes